sensor_msgs/Image depth
float32 fx
float32 fy
---
sensor_msgs/Image mask
